charts/index.yaml

```diff
@@ -3,7 +3,29 @@ entries:
   karmada:
   - apiVersion: v2
     appVersion: latest
-    created: "2023-08-03T16:11:27.348187800+08:00"
+    created: "2023-09-11T14:40:33.2057800-04:00"
+    dependencies:
+    - name: common
+      repository: https://charts.bitnami.com/bitnami
+      version: 1.x.x
+    description: A Helm chart for karmada
+    digest: 93a7051ab973c103411f74d7a516c673f64db447a4cdb1eae98efafff93a48b1
+    kubeVersion: '>= 1.16.0-0'
+    maintainers:
+    - email: jrkeen@hotmail.com
+      name: jrkeen
+    - email: jackson.cloudnative@gmail.com
+      name: pidb
+    - email: shentiecheng@huawei.com
+      name: Poor12
+    name: karmada
+    type: application
+    urls:
+    - https://github.com/karmada-io/karmada/releases/download/charts/karmada-chart-v1.7.0.tgz
+    version: v1.7.0
+  - apiVersion: v2
+    appVersion: latest
```

Comment:
hello, there seems to not be `lastet`, refer to below examples, it may be `v1.1.0`~ （tip: I don't know if I am correct, just  a reminder）

Reply:
What command do you use to update the index?

This is the command I executed:
```
helm repo index charts/karmada --url https://github.com/karmada-io/karmada/releases/download --merge charts/index.yaml
```

Reply:
> What command do you use to update the index?

sorry, I am not familiar with the `helm index` command, I just compared the indexes of previous versions and mistakenly thought that only the latest version's `appVersion` filed could be `lastet` value.

However, since the content is generated by your valid command, I think there is no problem~

Reply:
By the way, I wanna why after I executed the command exactly you provided, my `index.yaml` becomes:

![image](https://github.com/karmada-io/karmada/assets/30589999/186d2495-03d9-4cbe-b9e8-eb7fd38eda57)

Is there any mistake in my operation？

Reply:
```
-bash-5.0# helm repo index charts/karmada --url https://github.com/karmada-io/karmada/releases/download --merge charts/index.yaml
-bash-5.0# git status
On branch master
Your branch is up to date with 'origin/master'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	charts/karmada/index.yaml
```
The above command yields a new index file in `charts/karmada/` directory on my side.

Add seems just generated time changed in side: 
```diff
-bash-5.0# mv charts/karmada/index.yaml charts/index.yaml 
-bash-5.0# git diff 
diff --git a/charts/index.yaml b/charts/index.yaml
index 907a1d4f..349d871e 100644
--- a/charts/index.yaml
+++ b/charts/index.yaml
@@ -3,7 +3,7 @@ entries:
   karmada:
   - apiVersion: v2
     appVersion: latest
-    created: "2023-08-03T16:11:27.348187800+08:00"
+    created: "2023-08-03T16:11:27.3481878+08:00"
     dependencies:
     - name: common
       repository: https://charts.bitnami.com/bitnami
@@ -25,7 +25,7 @@ entries:
     version: v1.6.2
   - apiVersion: v2
     appVersion: v1.1.0
-    created: "2023-07-06T11:15:13.538429300+08:00"
+    created: "2023-07-06T11:15:13.5384293+08:00"
     dependencies:
     - name: common
       repository: https://charts.bitnami.com/bitnami
@@ -153,4 +153,4 @@ entries:
     urls:
     - https://github.com/karmada-io/karmada/releases/download/v1.2.0/karmada-chart-v1.2.0.tgz
     version: v1.2.0
-generated: "2023-08-08T09:58:58.100997700+08:00"
+generated: "2023-09-13T17:11:49.040502926+08:00"
```

> --url https://github.com/karmada-io/karmada/releases/download

Seems this is an invalid URL? What URL should provided here? 


By the way, I'm using the latest released version of `helm`: @v3.12.3
```
-bash-5.0# helm version
version.BuildInfo{Version:"v3.12.3", GitCommit:"3a31588ad33fe3b89af5a2a54ee1d25bfe6eaa5e", GitTreeState:"clean", GoVersion:"go1.20.7"}
```

Reply:
I think I have found the way:

1. download the `karmada-chart-v1.7.0.tgz` to dir `charts/karmada/`
2. run `helm repo index charts/karmada --url https://github.com/karmada-io/karmada/releases/download/v1.7.0 --merge charts/index.yaml`
3. run `mv charts/karmada/index.yaml charts/index.yaml`

Notes：

* `helm repo index charts/karmada` means helm will recurse the directory `charts/karmada` to find all the chart tgz file, it will generate index.yaml according to these valid chart tgz file.
* `--url` means the `entries.karmada[x].urls` filed in `index.yaml` while pinch param's value `https://github.com/karmada-io/karmada/releases/download/v1.7.0` and tgz chart file name `karmada-chart-v1.7.0.tgz` together as a final string
* `--merge charts/index.yaml` means the generated `index.yaml` will merge `charts/index.yaml`
* however, the final generated file lies in `charts/karmada/index.yaml`, so we need a `mv` command.

Reply:
Thank you, that worked for me.

As far as `latest` vs current version, we should update [this](https://github.com/karmada-io/karmada/blob/master/charts/karmada/Chart.yaml#L29) first, correct?

Reply:
> As far as latest vs current version, we should update [this](https://github.com/karmada-io/karmada/blob/master/charts/karmada/Chart.yaml#L29) first, correct?

Hello, in my opinion, we don't need to update that `appVersion`, one reason is this field is auto generated by final `Chart.yaml` in `karmada-chart-v1.7.0.tgz`, since we have released this chart tgz, we shall remain unchanged; another is that this filed didn't really used in our chart, no modification has no bad influence.

***

By the way, I am sorry to tell you that there may still be some problems with the `urls`, it is still `404`. 

The correct url should be `https://github.com/karmada-io/karmada/releases/download/v1.7.0/karmada-chart-v1.7.0.tgz` ~

Reply:
Unzippeing `https://github.com/karmada-io/karmada/releases/download/v1.7.0/karmada-chart-v1.7.0.tgz` still shows appVersion as latest. but I can create another Issue / PR for that since it's used in some of the labels `app.kubernetes.io/version`